haskell/test/Test/FA2/Transfer.hs

```diff
@@ -72,23 +72,18 @@ test_removed_position =
           , sppUpperTickIndex = upperTickIndex
           , sppLowerTickWitness = minTickIndex
           , sppUpperTickWitness = minTickIndex
-          , sppLiquidityDelta = liquidityDelta
-          , sppToX = owner
-          , sppToY = owner
+          , sppLiquidity = liquidityDelta
           , sppDeadline = deadline
           , sppMaximumTokensContributed = PerToken 1000000 1000000
           }
-      call cfmm (Call @"Set_position")
-        SetPositionParam
-          { sppLowerTickIndex = lowerTickIndex
-          , sppUpperTickIndex = upperTickIndex
-          , sppLowerTickWitness = minTickIndex
-          , sppUpperTickWitness = minTickIndex
-          , sppLiquidityDelta = -liquidityDelta
-          , sppToX = owner
-          , sppToY = owner
-          , sppDeadline = deadline
-          , sppMaximumTokensContributed = PerToken 1000000 1000000
+      call cfmm (Call @"Update_position")
+        UpdatePositionParam
+          { uppPositionId = PositionId 0
+          , uppLiquidityDelta = -(toInteger liquidityDelta)
+          , uppToX = owner
+          , uppToY = owner
+          , uppDeadline = deadline
+          , uppMaximumTokensContributed = PerToken 1000000 1000000
           }
 
```

Comment:
I think we should have a specific test in this module to check that transferring a position to an owner that already has one in that range will not result in issues (as it would have before)